source/assets/json/task_list_data.json

```diff
@@ -0,0 +1,20 @@
+[
+    {
+        "title": "First",
```

Comment:
Let's add an `id` field to each task too. That way we can uniquely refer to each item since two tasks might have the same name but they can't have the same id.

Reply:
We'll do this in a future task